dot/parachain/prospective-parachains/prospective_parachains.go

```diff
@@ -440,3 +445,86 @@ func (pp *ProspectiveParachains) answerProspectiveValidationDataRequest(
 		response <- nil
 	}
 }
+
+func (pp *ProspectiveParachains) answerHypotheticalMembershipRequest(
```

Comment:
```suggestion
func (pp *ProspectiveParachains) getHypotheticalMembership(
```
It's shorter and consistent with `getBackableCandidate()`.